File: JWT_Request.py
#!/usr/bin/python
import requests
import json

url       = "https://trefle.io/api/auth/claim?token=bWVXWThmZlpBUWlTV3NaaCtQWVM5QT09&origin=http://192.168.1.108"
response  = requests.request("POST", url).json()
JWT_Token = response['token']

print(JWT_Token) # This writes the token to the file www/JW_Token.txt because in the crontab I have > to file which means that what ever is outputed by this python script is writen to the file.

File: JWT_Request.js

function getJWT() {
    // curl -i -X POST "https://trefle.io/api/auth/claim?token=YOUR-TOKEN&origin=YOUR-WEBSITE-URL"
    var xhttp = new XMLHttpRequest();
    xhttp.open("POST", "https://trefle.io/api/auth/claim?token=bWVXWThmZlpBUWlTV3NaaCtQWVM5QT09&origin=192.168.1.254:5500/www/search.html")
    xhttp.send();
}
getJWT();